ldap_auth_provider.py

```diff
@@ -183,7 +183,7 @@ async def check_auth(
                 return None
 
             # Get full user id from localpart
-            user_id = self.account_handler.get_qualified_user_id(localpart)
+            user_id = self.account_handler.get_qualified_user_id(localpart.lower())
```

Comment:
I still think it'd be cleaner to do
`localpart = localpart.lower()`, above once

Reply:
Totally fair, I've changed it to reflect this.